Plugin/Magento/Checkout/CustomerData/Cart.php

```diff
@@ -75,9 +85,12 @@ public function __construct(
     public function afterGetSectionData(CustomerDataCart $subject, array $result)
     {
         $quote = $this->getQuote();
+        $result['quoteMaskedId'] = null;
+        $result['boltCartHints'] = null;
         if ($quote->getId()) {
             try {
                 $result['quoteMaskedId'] = $this->getQuoteMaskedId((int)$quote->getId());
+                $result['boltCartHints'] = $this->boltHelperCart->getHints($quote->getId(), self::HINTS_TYPE);
```

Comment:
any reason why we convert quote id to int in previous line but do not in this line?

Reply:
in method (i've created) getQuoteMaskedId i'm using strict declaration (getQuoteMaskedId(int $quoteId)), and for boltHelperCart->getHints(...) methods we don't have strict parameters.